src/HealthGPS.Console/csvparser.cpp

```diff
@@ -116,6 +116,7 @@ bool load_datatable_from_csv(hgps::core::DataTable &out_table, const poco::FileI
         throw std::runtime_error("Required columns not found in dataset.");
     }
 
+    hgps::core::DataTable out_table{};
```

Comment:
FYI `DataTable` should be default initialised here anyway. You don't need the extra braces.